source/utilities/server.ts

```diff
@@ -101,6 +111,7 @@ export const startServer = async (
       else address = details.address;
       const ip = getNetworkAddress();
 
+      const httpMode = useSsl ? 'https' : 'http';
```

Comment:
Could you rename this to `protocol`, since the variable is being used as the protocol of the server URL.

Reply:
Thanks for the feedback, this is done